llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1447,6 +1447,118 @@ LogicalResult TopkOp::getResultTilePosition(
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// PackOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult PackOp::verify() {
+  // input rank = r
+  // output rank = 2 * r (input rank + tile rank).
+  // inner tiles = r
```

Comment:
wrench: do you accept 0 to explicitly skip a dimension ?

Reply:
Do we really want that? Having `0` this way means it is pervasive in the code to check for `0` and do something else. Would a list of dimensions that are data tiled work?